README.md

```diff
@@ -104,7 +104,10 @@ Flags:
 ```
 
 **Single Command Launch**
-
+```
+$ simple-ec2 version
```

Comment:
Let's give this subcommand its own section in the README, not part of the `launch` subcommand documentation.

Reply:
Done